openpgp/packet/private_key.go

```diff
@@ -155,6 +156,56 @@ func mod64kHash(d []byte) uint16 {
 	return h
 }
 
+// This is the counterpart to the Decrypt() method below
+// @param passphase is the passphrase used to generate the derived-key based
```

Comment:
Generally we don't have doxygen-style comments in go (at least not in go-crypto). Could you try to rewrite it to something like this: https://github.com/keybase/go-crypto/blob/master/openpgp/write.go#L204-L208 ? It also describes that the arguments and return values but in more prose-like way. Then we will have everything consistent. Thank you!